### DISCUSSION TITLE
Inaccuracy of the translation of the Assistant "Mule" perk into Russian.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#15231] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

English description of the perk is:
"Stackable materials can have +2 items per stack in your inventory."
Used word "materials". This includes iron, copper, and other similar items.
In the Russian version the word "предметы" is used, this is a more general concept that includes both medicine and ammunition and many others, in essence, all stackable items.
As a result, there is a misunderstanding.
I don't see the point in carrying materials around in the inventory inside the submarine. There are crates and containers for that.
The perk is quite useless since ores does not belong to materials. 
Thus, the assistant can not carry more ores in an expedition.

### Reproduction steps

Obtain perk "mule"
Take in inventory anything that stacks except simple materials.
The stack size has not changed.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I passed this to the localization team, hopefully they'll address this in the next localization pass.

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/8ef1017c1ce425ffb08c5d7137f1b826ebd725df (commit in our private repo), fix coming up in the next update.
